### PR TITLE
Start spinning Nightly build workers when the trigger exists.

### PR DESCRIPTION
**Pull request by nakhbari**

### CODE REVIEW

File: services/worker_service.rb [hunk 1]
```diff
@@ -8,44 +9,61 @@ module FastlaneCI
...
+      self.project_to_workers_dictionary[workers_key] = nil
+    end
+
+    def num_workers
+      self.project_to_workers_dictionary.values.reduce(0) do |sum, workers|
+        sum + workers.length
+      end
+    end
+
+    def project_has_trigger_type(project: nil, trigger_type: nil)
```
Comment (snatchev) [inline services/worker_service.rb]: Methods that return booleans should end in `?`

File: services/worker_service.rb [hunk 2]
```diff
@@ -8,44 +9,61 @@ module FastlaneCI
...
+      raise "Need both project and trigger_type" unless project && trigger_type
```
Comment (snatchev) [inline services/worker_service.rb]: Did we settle on a minimum ruby version? If it was > 2.2 then declaring the named parameters as `(project:, trigger_type:)` will make them required by runtime.

File: services/worker_service.rb (hunk 1 above)
Comment (KrauseFx) [reply services/worker_service.rb]: Nice catch 👍

File: services/worker_service.rb (hunk 2 above)
Comment (nakhbari) [reply services/worker_service.rb]: Cool! i'm not sure if we have a minimum ruby version. @KrauseFx is there a minimum ruby version set?

File: services/worker_service.rb (hunk 1 above)
Comment (nakhbari) [reply services/worker_service.rb]: fixed.

File: services/worker_service.rb (hunk 2 above)
Comment (taquitos) [reply services/worker_service.rb]: 2.3.3p222 🎉

**Pull request merged**